.github/workflows/build.yml

```diff
@@ -0,0 +1,165 @@
+name: cesium-native
+on: [push, pull_request]
+jobs:
+  QuickChecks:
+    name: "Quick Checks"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Check source formatting
+        run: |
+          npm install
+          npm run format -- --dry-run -Werror
+  Documentation:
```

Comment:
I appreciate this single documentation step as opposed to the older version which had it with each build.